docs/resources/sso_settings.md

```diff
@@ -72,17 +72,87 @@ resource "grafana_sso_settings" "saml_sso_settings" {
 
 ### Required
 
-- `provider_name` (String) The name of the SSO provider. Supported values: github, gitlab, google, azuread, okta, generic_oauth, saml.
+- `provider_name` (String) The name of the SSO provider. Supported values: github, gitlab, google, azuread, okta, generic_oauth, saml, ldap.
 
 ### Optional
 
+- `ldap_settings` (Block Set, Max: 1) The LDAP settings set. Required for the ldap provider. (see [below for nested schema](#nestedblock--ldap_settings))
```

Comment:
```suggestion
- `ldap_settings` (Block Set, Max: 1) The LDAP settings set. Required for the LDAP provider. (see [below for nested schema](#nestedblock--ldap_settings))
```

Reply:
I wanted to keep `ldap` in lowercase here to match the value of the `provider_name` property. It's the same for the other providers (look at the saml description).

I understand it's a bit confusing to see `LDAP` in uppercase in the first sentence and then `ldap` in the second sentence. But the first one refers to the protocol name, while the second one should be the value users have to put in the `provider_name`. WDYT?

Reply:
Sounds reasonable, thanks!